tests/debugging/test_debugger.py

```diff
@@ -1138,3 +1138,38 @@ def test_debugger_redacted_identifiers():
             },
             "throwable": None,
         }
+
+
+def test_debugger_exception_conditional_function_probe():
+    """
+    Test that we can have a condition on the exception on a function probe when
+    the condition is evaluated on exit.
+    """
+    from tests.submod import stuff
+
+    snapshots = simple_debugger_test(
+        create_snapshot_function_probe(
+            probe_id="probe-instance-method",
+            module="tests.submod.stuff",
+            func_qname="throwexcstuff",
+            evaluate_at=ProbeEvaluateTimingForMethod.EXIT,
+            condition=DDExpression(
+                dsl="expr.__class__.__name__ == 'Expression'",
```

Comment:
can we also test this with isUndefined(@excpetion) and instranceof(@exception,'Exception) (not sure the syntax and if RFC is completed around this one)

Reply:
🤔 I don't think we have `instanceof` in the DSL?

Reply:
We can add the `isUndefined` test when we make the change to `isDefined`. WDYT?